quipucords/api/credential/serializer.py

```diff
@@ -47,6 +47,8 @@ class CredentialSerializer(NotEmptySerializer):
     ssh_passphrase = CharField(required=False, max_length=1024,
                                allow_null=True,
                                style={'input_type': 'password'})
+    become_method = CharField(required=False, max_length=6, allow_null=True,
+                              style={'input_type': 'become_method'})
```

Comment:
become_method should use choices like cred_type. Also no style is necessary, plus I don't think what you have does anything.

`style={'input_type': 'password'}` tells the Django browsable API to hide the value when viewing in the browser. `become_method` isn't anything.